harness/determined/cli/config_policies.py

```diff
@@ -24,36 +36,65 @@ def describe_config_policies(args: argparse.Namespace) -> None:
 
 def set_config_policies(args: argparse.Namespace) -> None:
     sess = cli.setup_session(args)
+    if not args.workspace_name and not args.glob:
+         raise api.errors.BadRequestException(
+            "must provide either --workspace-name WORKSPACE_NAME or --glob"
```

Comment:
I think we just want either/or. If workspace name is given, we use it. Otherwise, we assume global.

Reply:
Done!